test/unit/pr_checker.test.js

```diff
@@ -307,6 +308,42 @@ describe('PRChecker', () => {
       assert(!status);
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
+
+    it('should only log last three commits if multiple', () => {
+      const logger = new TestLogger();
+      const { commits, comment } = mulipleCommitsAfterCi;
+
+      const expectedLogs = {
+        warn: [
+          [ 'Commits pushed after the last Full CI run:' ],
+          [ 'last 3 commits...' ],
```

Comment:
Can you merge this line with the line above? `Commits pushed after the last Full CI run. The last 3 commits are..`

Reply:
@joyeecheung i will fix it.

Reply:
done